diesel/src/pg/types/ranges.rs

```diff
@@ -88,7 +89,9 @@ where
             let elem_size = bytes.read_i32::<NetworkEndian>()?;
             let (elem_bytes, new_bytes) = bytes.split_at(elem_size as usize);
             bytes = new_bytes;
-            let value = T::from_sql(Some(elem_bytes))?;
+            let value = T::from_sql(Some(
+                PgValue::with_oid(elem_bytes.as_ptr() as *mut u8, 1, 0) // TODO Find OID
```

Comment:
Basically the same as for `PgInterval` implementation above: We need to find a way to use `HasSqlType::metadata` here.